src/components/LanguageMenu/LanguageNativeSelect.tsx

```diff
@@ -18,23 +18,32 @@ export default function LanguageNativeSelect() {
 
   const changeLanguageHandler = React.useCallback(async (lang: string) => {
     setLang(lang);
+    localStorage.setItem("i18nextLng", lang);
```

Comment:
In this project we're using basically SessionStorage. Why use LocalStorage?
Remember: every new data stored in Local/session/cookie *must* be declared in the cookie policy

Reply:
I chose localStorage because it preserves the language across multiple sessions. This way, a foreign user won't have to select the language each time they navigate through checkout.